Source/Core/Common/NandPaths.cpp

```diff
@@ -151,4 +152,12 @@ std::string UnescapeFileName(const std::string& filename)
 
   return result;
 }
+
+bool IsFileNameSafe(const std::string& filename)
```

Comment:
That would be inconsistent with the other functions. Should we also change those? Seems out of scope for this PR.

Reply:
Given this is just inspecting the string rather than performing behavior in the string data or making a copy of the data, a view is perfectly fine here IMO in relation to the other functions.

Reply:
Ah alright, fair enough.